css/properties/offset-position.json

```diff
@@ -7,14 +7,27 @@
           "spec_url": "https://drafts.fxtf.org/motion/#offset-position-property",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "See <a href='https://crbug.com/638055'>bug 638055</a>."
+              "version_added": "115",
+              "notes": "The keyword `normal` is not part of the syntax.",
```

Comment:
Rather than add a note to Chrome, we should create a subfeature for this value, especially as Firefox supports it.